contracts/adapters/GmpTransferAdapter.sol

```diff
@@ -0,0 +1,114 @@
+// The Licensed Work is (c) 2022 Sygma
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.11;
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+import "../../contracts/interfaces/IBridge.sol";
+import "../../contracts/interfaces/IFeeHandler.sol";
+import "../XERC20/interfaces/IXERC20.sol";
+
+contract GmpTransferAdapter is AccessControl {
```

Comment:
Are we missing liquidity withdrawal function?
I see some unused events for it.

Reply:
There is no liquidity on adapter, tokens that are deposited are XERC20 and always get minted/burned